docs/api-reference/index.md

```diff
@@ -81,6 +81,8 @@
    widgets.drawing.DrawingTool
    widgets.drawing.PointsTool
 
+   widgets.clip3d.Clip3dTool
+   widgets.clip3d.ClippingPlanes
    widgets.cut3d.Cut3dTool
```

Comment:
Yes, it's the old tool.
I was thinking of keeping it in case someone was using it.
But you're right, I should probably deprecate it, or at least indicate that we will not be maintaining it.

The alternative is of course just to remove it...

Reply:
Either works for me, I think it's best you decide